internal/input/input_test.go

```diff
@@ -0,0 +1,34 @@
+package input
+
+import (
+	"fmt"
+	"testing"
+)
+
+func TestStrSize2ByteSize(T *testing.T) {
+
+	data := []struct {
```

Comment:
I love table driven tests. Nice work on these!

One thing I like to do is add a `name` property to the struct here so that you can name your tests and in the subtest `t.Run`, you can use `data.name` so that test results are a little clearer.

I also prefer to use the expanded struct assignment syntax for each test case. So `{"100kb", 100, 102400}` would become:

```
{
  in: "100kb",
  inTotalSize: 100,
  out: 102400,
}
```

IMO, this makes it a little bit simpler to visually parse what the test and expectations are.

Reply:
Done, nice idea, the expanded struct style i felt like typing a lot, but for sure it increases the readability ;)

Reply:
It's certainly a bit more verbose, but clarity should take precedence over brevity in my opinion